src/customer-stories.njk

```diff
@@ -0,0 +1,77 @@
+---
+layout: nohero
+sitemapPriority: 0.8
+meta:
+    title: Customer Stories
+---
+<div class="container m-auto text-left max-w-4xl pb-24 w-full">
+    <div class="px-2">
+        <h1>Customer Stories</h1>
+    </div>
+    {% if collections.posts.length > 0 %}
+    <ul class="flex flex-wrap">
+        {%- for item in collections.stories | reverse -%}
+        {% if loop.first %}
+        <li class="w-full md:w-2/3 mt-2 px-2 pb-4">
+            <a href="{{ item.url }}" class="w-full flex flex-col group hover:no-underline">
+                <div class="pr-2">
+                    {% if item.data.image %}
+                        <div class="w-full h-60 md:h-72">
+                            {% set imageSrc = [item.data.image ] | join %}
+                            {% set imageDescription = ["Image representing ", item.data.title] | join %}
+                            <img src="{{ imageSrc }}" alt="{{ imageDescription }}" class="object-cover w-full h-full" />
```

Comment:
@Pezmc thoughts on this please? The shortcode doesn't allow us to pass CSS classes to it, here we want to make the most of Tailwind's `object-cover`. We could switch the layout to be using a background-image instead, but feles excessive.

Reply:
@joepavitt Can you apply the CSS directly? As it stands this image won't go through the image pipeline and get resized at all.

``css
.object-cover {
    object-fit: cover;
}
```

Reply:
yes, but on what? the `image` shortcode doesn't let us apply styles either, right?

Reply:
```
 <div class="w-full h-60 md:h-72 image-cover">
   {{ image-tag }}
 </div>
 ```
 
 ```css
.image-cover > img {
    object-fit: cover;
}
```

Reply:
oh that's dirty - it'll work though